ntoskrnl/cc/view.c

```diff
@@ -628,6 +628,7 @@ CcRosLookupVacb (
     PLIST_ENTRY current_entry;
     PROS_VACB current;
     KIRQL oldIrql;
+    PROS_VACB Ret = NULL;
```

Comment:
```suggestion
    PROS_VACB current, Ret = NULL;
    KIRQL oldIrql;
```

Reply:
I think the current style is more readable.